src/Playwright.Tests/TracingTests.cs

```diff
@@ -125,6 +125,57 @@ public async Task ShouldCollectTwoTraces()
 
         }
 
+        [PlaywrightTest("tracing.spec.ts", "should work with multiple chunks")]
+        public async Task ShouldWorkWithMultipleChunks()
+        {
+            using var tmp = new TempDirectory();
+
+            await Context.Tracing.StartAsync(new()
+            {
+                Screenshots = true,
+                Snapshots = true,
+            });
+
+            var page = await Context.NewPageAsync();
+            await page.GotoAsync(Server.Prefix + "/frames/frame.html");
+
+            await Context.Tracing.StartChunkAsync();
+            await page.SetContentAsync("<button>Click</button>");
+            await page.ClickAsync("'Click'");
+            var traceFile1 = Path.Combine(tmp.Path, "trace1.zip");
+            await Context.Tracing.StopChunkAsync(new TracingStopChunkOptions { Path = traceFile1 });
+
+            await Context.Tracing.StartChunkAsync();
+            await page.HoverAsync("'Click'");
+            var traceFile2 = Path.Combine(tmp.Path, "trace2.zip");
+            await Context.Tracing.StopChunkAsync(new TracingStopChunkOptions { Path = traceFile2 });
+
+            await Context.Tracing.StartChunkAsync();
+            await page.HoverAsync("'Click'");
+            await Context.Tracing.StopChunkAsync(); // Should stop without a path.
+
+            {
+                var (events, resources) = ParseTrace(traceFile1);
+                Assert.AreEqual("context-options", events[0].Type);
+                Assert.GreaterOrEqual(events.Where(x => x.Metadata?.ApiName == "Page.SetContentAsync").Count(), 1);
```

Comment:
can we use `GetActions` there to assert it? since line 161 and 162 are the same. Like we have upstream.